app/src/main/res/values/strings.xml

```diff
@@ -415,6 +416,7 @@ How to translate with transifex:
     <string name="nc_sent_poll_you">You sent a poll.</string>
     <string name="nc_sent_location_you">You sent a location.</string>
     <string name="nc_sent_voice_you">You sent a voice message.</string>
+    <string name="nc_sent_deck_card_you">You: %1$s</string>
```

Comment:
You sent a deck card.

Reply:
@mahibi - The string is modified to align with web

Reply:
The web is always saying `You: …` but that's not the best. Use what Marcel suggested, to align with the other messages of the same type that exist atm in Android